cli/fathom_web/vectorizer.py

```diff
@@ -190,7 +189,15 @@ def fathom_fox_addon(ruleset_file):
         # It should be okay to reference geckodriver outside the
         # locked_cached_fathom() lock, since that part of the cache is
         # immutable:
-        yield addon_path, (fathom_fox / 'node_modules' / '.bin' / 'geckodriver')
+        yield addon_path, add_cmd_if_windows(fathom_fox / 'node_modules' / '.bin' / 'geckodriver')
+
+
+def add_cmd_if_windows(binary_path):
+    """Add the .cmd suffix to a pathlib.Path if running on Windows."""
+    if platform.system() == 'Windows':
```

Comment:
You're still running Cygwin, correct? And you still get `.cmd` files laid down? (I ask in case we should instead be checking for `sys.platform == 'win32'`.)

Reply:
I do have Cygwin around, but I'm not in a Cygwin terminal to run this (or anything). It's mostly there to provide the utilities the Makefile uses. Regardless, I think you're right that we should be checking for `sys.platform`.